### DISCUSSION TITLE
Welcome to komapper Discussions!

### DISCUSSION THREAD

**Discussion by nakamura-to** [komapper/komapper#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lorefnon** [comment]
Hello @nakamura-to 

I am evaluating a kotlin orm for a new project, and found this through kotlin slack channel. I am curious why you maintain two similar projects - komapper & doma and if one of them was more recommended for future use.

**Reply by nakamura-to** [reply]
Hi, @lorefnon 

Thank you for your interest in Komapper.

My answer is simple. Doma is a library for Java, and Komapper is a library for Kotlin. To add, Doma only supports JDBC, but Komapper supports R2DBC as well as JDBC. There is no plan to support R2DBC in Doma.

If you are going to use Kotlin in your project, I recommend Komapper.

**Reply by lorefnon** [reply]
I see. Thanks for your reply and for creating this library - the API looks quite friendly. Use of KSP is also a big plus for me.
